src/components/pages/metadata-api/metadata-api-presentation.tsx

```diff
@@ -0,0 +1,337 @@
+                alt="A bird flying over a stack of documents"
+              />
+            </div>
+          </div>
+        </section>
+
+        <section
+          className={clsx('bg-orangeBow bg-100% px-2 mt-0 !pt-16 mb-20')}
+        >
+          <div className="mt-2 pb-16 sm:flex text-center text-xl max-w-4xl mx-auto px-4">
+            <div className="flex-1 mt-5">
+              <b className="text-brand font-handwritten text-4xl">20,000+</b>
+              <br />
+              Educational resources have been created with the{' '}
+              <Link href="https://serlo.org/editor">Serlo Editor</Link>
+            </div>
+            <div className="flex-1 mt-5">
+              <b className="text-brand font-handwritten text-4xl">1 Mio+</b>
+              <br />
+              Users per month already learn with Serlo
+            </div>
+          </div>
+        </section>
+
+        <section className={clsx('mt-24 pb-16')}>
+          <div className="text-3xl leading-cozy max-w-4xl text-center mx-auto"></div>
+
+          <div className="mt-12 text-3xl leading-cozy max-w-4xl text-center mx-auto">
+            <h2 className="text-center text-4xl leading-cozy tracking-tight font-extrabold mb-4">
+              Development and license
+            </h2>
+            <div className="text-left mx-side">
+              <p className="mt-4 text-xl leading-cozy flex-1">
+                Join our growing community by integrating our content through
+                the Serlo Metadata API into your apps and change the future of
+                education with us!
+              </p>
+              <p className="mt-2 text-xl leading-cozy">
+                Licensed under,{' '}
+                <Link href="https://creativecommons.org/licenses/by-sa/4.0/">
+                  CC-BY-SA 4.0
+                </Link>
+                , the usage is completely free of cost.
+              </p>
+              <ul className="serlo-ul text-xl">
+                <li>
+                  Retrieve metadata of all our articles, videos, courses and
+                  quizzes
+                </li>
+                <li>Easy to use interface</li>
+                <li>
+                  Follow popular aggregators such as{' '}
+                  <Link href="https://wirlernenonline.de/">
+                    WirLernenOnline
+                  </Link>{' '}
+                  that have already integrated our content
+                </li>
+                <li>
+                  Fully compliant with the state of the art{' '}
+                  <Link href="https://dini-ag-kim.github.io/amb/draft/">
+                    AMB standard
+                  </Link>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </section>
+        <CodeSnippets />
+        <div className="mt-4" />
+        <ExampleResponse />
+      </div>
+    </>
+  )
+}
+
+const codeSnippets: Record<string, string> = {
```

Comment:
As said, these code snippets would have to be changed in order to really work. 
But my suggestion is to introduce a simple 'Docs' link, which would redirect to a Github Wiki page, where devs could get a fully explanation on how to use our GraphQL query.